prismic-model/js/manual-promos.js

```diff
@@ -0,0 +1,15 @@
+import structuredText from './parts/structured-text';
+import image from './parts/image';
+import link from './parts/link';
+
+const ManualPromo = {
```

Comment:
Could we name this differently. We've all but killed the name promo.

What about just a `Card`. You could prefix it with something like `SimpleCard` - but I might leave that to the rendering as the data is the same.

Having common naming here will also help with using parsing that we have already written.